packages/codemirror/src/Editor.ts

```diff
@@ -51,6 +51,7 @@ export class Editor extends HTMLWidget {
         this._codemirror.setGutterMarker(line, "CodeMirror-guttermarker", marker);
         marker.onmouseenter = onmouseenter;
         marker.onmouseleave = onmouseleave;
+        marker.onclick = onclick;
```

Comment:
@jbrundage Does onClick receive any arguments?

Reply:
It does. The PointerEvent is the first argument.